.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Settings common to each job
+var_job_defaults: &job_defaults
+  working_directory: ~/verdaccio-google-cloud-storage
+  docker:
+    - image: cimg/node:lts-browsers
+
+orbs:
+  node: circleci/node@5.1.0
+  build-tools: circleci/build-tools@3.0.0
+
+commands:
+  # Command for checking out the source code from GitHub. This also ensures that the source code
+  # can be merged to the master branch without conflicts.
+  checkout_and_rebase:
+    description: Checkout and verify clean merge with master
+    steps:
+      - checkout
+      - build-tools/merge-with-parent:
+          parent: master
+
+  setup:
+    description: 'Setup Executor'
```

Comment:
The description sounds confusing (not sure what "executor" refers to here).

Reply:
Changed to `Download the workspace content from the cache`